sw/device/tests/csrng_edn_concurrency_test.c

```diff
@@ -209,14 +209,11 @@ static void otbn_task(void *task_parameters) {
     LOG_INFO("OTBN:START");
     for (size_t i = 0; i < task_iter_count_max[kTestTaskIdOtbn]; ++i) {
       otbn_randomness_test_start(&otbn);
-      bool is_busy = true;
-      while (is_busy) {
-        CHECK(otbn_busy_check(&otbn, &is_busy) == kOtbnOk);
-        if (!is_busy) {
-          break;
-        }
+      dif_otbn_status_t status;
+      do {
+        CHECK_DIF_OK(dif_otbn_get_status(&otbn, &status));
         ottf_task_yield();
-      }
+      } while (status != kDifOtbnStatusIdle && status != kDifOtbnStatusLocked);
       CHECK(otbn_randomness_test_end(&otbn, /*skip_otbn_done_check=*/false));
```

Comment:
nit: do you think `otbn_yield_wait_for_done(otbn)` would be useful? (if we have more loops like this)

Reply:
It looks like this is the only occurrence, so I'll leave it inlined for now.